### ISSUE TITLE
Nightly fluid-tests take too long

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#3512]
they are getting aborted:
~~~Running tests for FluidDynamicsApplication
........................................./home/ubuntu/Kratos/applications//FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

ABORT: Tests for FluidDynamicsApplication took to long. Process Killed.
~~~

**Comment by jcotela** [comment]
Apparently, the Darcy tests take too long to run now, which is weird because they are relatively old ones... @ddiezrod do you think that this could be related to some of the changes in the TwoFluid element?

**Comment by jcotela** [comment]
Update: the tests stopped working between 14 nov and 19 nov (I have no nightly emails from the dates in between), this is the list of PR merged between these two dates: https://github.com/KratosMultiphysics/Kratos/pulls?page=1&q=is%3Apr+is%3Aclosed+merged%3A2018-11-14..2018-11-19+base%3Amaster&utf8=%E2%9C%93

Looks like the unification of the fluid solvers was in that date range, so maybe we inadvertently changed the settings for some tests?

**Comment by jcotela** [comment]
I hope this will be solved now that #3616 is merged... Let's see what the next test report says.

**Comment by jcotela** [comment]
Looks like this was not solved by #3616, only 41 of 45 (or so) tests were run today. @KratosMultiphysics/fluid-dynamics we need to see which ones we can cut.

**Comment by jcotela** [comment]
Nightly tests ran in time tonight after merging #4117, closing.
